config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
On the blog [homepage](https://ipfs.io/ipfs/QmdSwmRVWDDpDoBYXJcjHavF1jFhvkvaWwGAXeqAQjS1ii/) (and on the [post list page](https://ipfs.io/ipfs/QmdSwmRVWDDpDoBYXJcjHavF1jFhvkvaWwGAXeqAQjS1ii/post/) which is unused) the URLs to blog posts are now relative to the current page, so wherever the site is deployed these links will still work.

I changed the hugo variable `.Permalink` to `.URL` on these pages to do this. I didn't realise permalinks were being used here!

The RSS template is still using `.Permalink` so it gets the full URL including the defined `baseURL`.

FWIW, I think this is the right way to do it, if you ask for a permalink you should get one, but you need to define `baseURL` beforehand because hugo doesn't know where you're going to deploy your website.